invenio_records_files/jsonschemas/records-files/records-files-v1.0.0.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "title": "File schema.",
+    "description": "Describe information needed for files in records.",
+    "additionalProperties": false,
```

Comment:
The schema will be easier to reuse if we don't set ``"additionalProperties": false,``

On Zenodo e.g. I'd like to add a ``type`` and ``previewer`` property.